scripts/dev.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
```

Comment:
This should live in the cli folder since it's not global to the mono repo.

Reply:
Done!